setup.sh

```diff
@@ -138,7 +203,8 @@ function package ()
 
 function release ()
 {
-  runUnitTests
+  travis
```

Comment:
Make this function to start with a verb?  It would be so much more clear when I read how it's used here.